src/jdk.compiler/share/man/javac.1

```diff
@@ -636,6 +636,11 @@ Specifies the format for documentation comments.
 .RS
 .RE
 .TP
+.B \f[CB]\-\-help\-lint\f[R]
+Print the supported keys of option \-Xlint.
+.RS
+.RE
+.TP
```

Comment:
This change is somewhat problematic. The nroff man files are generated from Markdown files which are not (yet) open source. Don't ask: long story; we're working on it.

The bottom line is that this change should really be made to the upstream Markdown file.  Direct changes to this file are at risk of being overwritten at some point in the future.